pkg/store/google_secret_manager_store.go

```
@@ -0,0 +1,248 @@
+package store
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	secretmanagerpb "cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)